src/App.tsx

```diff
@@ -15,36 +17,82 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+const enum SortType {
+  Alphabet = 'Alphabet',
+  Length = 'Length',
+  Default = '',
+}
+
+const normalizeGoods = (goods: string[], type: SortType, reversed: boolean) => {
+  const compareGoods = (good1: string, good2: string) => {
+    switch (type) {
+      case SortType.Alphabet:
+        return good1.localeCompare(good2);
+      case SortType.Length:
+        return good1.length - good2.length;
+      default:
+        return 0;
+    }
+  };
+
+  const sortedGoods = [...goods].sort(compareGoods);
+
+  return reversed ? sortedGoods.reverse() : sortedGoods;
+};
+
 export const App: React.FC = () => {
+  const [sortType, setSortType] = useState<SortType>(SortType.Default);
+  const [reversed, setReversed] = useState(false);
```

Comment:
```suggestion
  const [isReversed, setIsReversed] = useState(false);
```
Boolean variables should have prefix `is` or `has`